src/args.py

```diff
@@ -21,27 +30,34 @@
 a = [7, 6, 5, 4]
 
 # What thing do you have to add to make this work?
-print(f2(a))    # Should print 22
+print(f2(*a))   # Should print 22
 
 # Write a function f3 that accepts either one or two arguments. If one argument,
 # it returns that value plus 1. If two arguments, it returns the sum of the
 # arguments. Google "python default arguments" for a hint.
 
 # YOUR CODE HERE
+def f3(int1, int2 = 1):
+    return int(int1) + int(int2)
+
 
 print(f3(1, 2))  # Should print 3
 print(f3(8))     # Should print 9
 
 
 # Write a function f4 that accepts an arbitrary number of keyword arguments and
-# prints ouf the keys and values like so:
+# prints out the keys and values like so:
 #
 # key: foo, value: bar
 # key: baz, value: 12
 #
 # Google "python keyword arguments".
 
 # YOUR CODE HERE
+def f4(**kwargs):
+    print(kwargs)
+    for value in kwargs:
```

Comment:
I'll give you a hint here. kwargs is taking in an object with both **keys** and  **values**
take a look at this resource and lemme know if you need any help
https://www.digitalocean.com/community/tutorials/how-to-use-args-and-kwargs-in-python-3